From 341b37d5e7911171e1b93cbb93c0005cbe7da805 Mon Sep 17 00:00:00 2001
From: Andreas Baierl <ichgehimkreisrum.de>
Date: Wed, 24 Oct 2018 10:00:29 +0200
Subject: [PATCH 41/47] lima/gpir: Rename gpir_lower_funcs

In preparation for future lower functions we make clear, that the existing
ones are executed after gpir_reduce_reg_pressure_prog.

Signed-off-by: Andreas Baierl <ichgeh@imkreisrum.de>
---
 src/gallium/drivers/lima/ir/gp/lower.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gallium/drivers/lima/ir/gp/lower.c b/src/gallium/drivers/lima/ir/gp/lower.c
index 28b619b..f450849 100644
--- a/src/gallium/drivers/lima/ir/gp/lower.c
+++ b/src/gallium/drivers/lima/ir/gp/lower.c
@@ -443,7 +443,7 @@ static bool gpir_lower_abs(gpir_block *block, gpir_node *node)
    return true;
 }
 
-static bool (*gpir_lower_funcs[gpir_op_num])(gpir_block *, gpir_node *) = {
+static bool (*gpir_post_rsched_lower_funcs[gpir_op_num])(gpir_block *, gpir_node *) = {
    [gpir_op_neg] = gpir_lower_neg,
    [gpir_op_rcp] = gpir_lower_complex,
    [gpir_op_rsqrt] = gpir_lower_complex,
@@ -472,8 +472,8 @@ bool gpir_post_rsched_lower_prog(gpir_compiler *comp)
 {
    list_for_each_entry(gpir_block, block, &comp->block_list, list) {
       list_for_each_entry_safe(gpir_node, node, &block->node_list, list) {
-         if (gpir_lower_funcs[node->op] &&
-             !gpir_lower_funcs[node->op](block, node))
+         if (gpir_post_rsched_lower_funcs[node->op] &&
+             !gpir_post_rsched_lower_funcs[node->op](block, node))
             return false;
       }
    }
-- 
2.0.1

